inception-doc/src/main/resources/META-INF/asciidoc/developer-guide/release.adoc

```diff
@@ -0,0 +1,101 @@
+= Releasing INCEpTION
+
+== Prerequisites
+
+When deploying to an Artifactory instance, make sure that the UKP Maven Repository is with your
+user credentials are added to your machines `settings.xml`. In our case, the repository
+`ukp-oss-releases` has to be listed with a user and password that belongs to the `trusted-user`
+group. See https://dkpro.github.io/dkpro-core/pages/setup-maven[DKPro Core - Setting up Maven repository]
+for more information on how to do this.
+
+== Release steps
+
+[arabic]
+. Create a release issue in the INCEpTION Github issue tracker
+. Merge the branch you want to release (e.g. `{revnumber}`) into the
+  master branch. This can be done right away with push permissions to
+  master or requires a pull request.
+. Checkout the branch you want to release. Create a new branch `released_branch` and check
+  it out, call it e.g. `release/issue-number-release-{revnumber}`
+. Run `mvn -Dmaven.surefire.heap=6g -Darguments="-Dmaven.surefire.heap=6g" -DautoVersionSubmodules release:prepare release:perform`;
+  the released jar is then in `target/checkout/inception-app-webapp/target/`, look for the standalone jar
+. Immediately after the release, again merge the released branch into the
+  master branch, using `git merge -s ours `released_branch`. Create a
+  pull request for this. Doing this second merge does not actually change
```

Comment:
Well, it *does* change things if there are bugfixes to the maintenance branch which have not yet been merged into master.

Reply:
Thats why we merged stable into master before, didnt we?

Reply:
Yes, but if I read correctly, there was no PR the first time - hence all the changes are included in this PR.

Reply:
Ok, I get it now: when you say there are no changes, you talk specifically about the **second merge**, not about the entire PR to master.